app/package.json

```diff
@@ -11,7 +10,14 @@
     "build-dev": "webpack --mode development",
     "start": "webpack-dev-server --mode development",
     "start-build-time-profile": "webpack-dev-server --mode development --profile --progress",
-    "prod-server": "npx http-server -p 3000 --proxy http://localhost:3000\\? dist"
+    "prod-server": "npx http-server -p 3000 --proxy http://localhost:3000\\? dist",
+    "dev": "electron-webpack dev",
+    "compile": "electron-webpack",
+    "dist": "yarn compile && electron-builder",
+    "dist:dir": "yarn dist --dir -c.compression=store -c.mac.identity=null"
+  },
+  "dependencies": {
+    "source-map-support": "^0.5.12"
```

Comment:
do we need this in deps? I'm trying to keep deps as small as possible (zero) in order to keep binary small.

Reply:
@wdanilo it seems to be part of Electron's loader. I couldn't workaround it. More research is needed.

Reply:
Can you provide some sources for that? Where do you know its part of electron loader and what is electron loader exactly?